src/exo/API_scheduling.py

```diff
@@ -1253,6 +1423,10 @@ def expand_dim(proc, buf_cursor, alloc_dim, indexing_expr):
                           of the new buffer dimension.
         indexing_expr   - (string) an expression to index the newly
                           created dimension with.
+        rc              - (bool) whether to return relevant cursors
+
+    relevant cursors:
+        alloc           - a cursor to the modifed alloc
 
```

Comment:
Not pertinent to this PR so much, but there is a TODO to rename this op.  I agree with `add_dim` except furthermore suggest `add_dims` since the plural makes it more clear that you're talking about the dimensions in aggregate rather than modifying an individual dimension.